lib/commands/general.js

```diff
@@ -92,11 +102,24 @@ export async function removeApp(appId) {
 /**
  * @this RokuDriver
  * @param {string} appId
+ * @param {string?} contentId
+ * @param {string?} mediaType
```

Comment:
These mean `string|null`; the function must be called with three (3) parameters, and they cannot be `undefined`. This means `string|undefined`:

```js
/**
  * @param {string} [contentId]
  * @param {string} [mediaType]
  */
```

Just making sure this is what you want.

Also might as well add descriptions for these, as we'll need to do that eventually.